docs/deployments/kubernetes/prerequisites.mdx

```diff
@@ -12,7 +12,11 @@ import TabItem from '@theme/TabItem';
 
 # Prerequisites
 
-## Access to the Docker and Helm Registry
+## Helm version
+
+Ensure you are using Helm version 3.16.0
```

Comment:
3.16.0 is only a minimum version contstaint. It can be helm version >3.16.x but <4.x.